src/sentry/tasks/post_process.py

```diff
@@ -437,6 +438,16 @@ def post_process_group(
                 plugin_post_process_group(
                     plugin_slug=plugin.slug, event=event, is_new=is_new, is_regresion=is_regression
                 )
+            if features.has(
+                "organizations:sentry-functions",
+                event.project.organization,
+            ):
```

Comment:
We generally try to avoid using feature flags that are hooked up to flagr in `post_process_group` and other high volume places. flagr can arbitrarily error and cause the rest of processing to fail. As well as that, if we're timing out, this could cause post_process requests to back up.

Ideally we'd use a non-flagr flag here, and we should also wrap all of this code in a try/catch so that we don't risk breaking `post_process_group`